### PR TITLE
[CLEANUP] Drop support for Node v16

### PR DESCRIPTION
**Pull request by Shishouille**
Related to the issue #10384

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -93,7 +93,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```
Comment (kategengler) [inline .github/workflows/ci.yml:96]:
```suggestion
          node-version: 18.17
```
18.18 apparently has problems with embroider

**Review by mansona — APPROVED** [review]
looks good to me 🎉 Glad that CI is passing again!

**Comment by kategengler** [comment]
Thank you!

**Comment by Shishouille** [comment]
Wow! Nice catch @kategengler :) Thanks!

**Pull request merged**